src/os/resourcesdir.zig

```diff
@@ -21,7 +21,16 @@ pub fn resourcesDir(alloc: std.mem.Allocator) !?[]const u8 {
 
     // This is the sentinel value we look for in the path to know
     // we've found the resources directory.
-    const sentinel = "terminfo/ghostty.termcap";
+    const sentinels = switch (comptime builtin.target.os.tag) {
```

Comment:
Why was introducing multiple necessary? Shouldn't a single work?

I understand the platform difference just not why this is now an array.

Reply:
Belt and suspenders I guess. I've switched back to a single file.